dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -183,6 +185,22 @@ public Entry addAliases(@ApiParam(hidden = true) @Parameter(hidden = true, name
         return verifiedVersions;
     }
 
+    @GET
+    @Path("/{entryId}/versions")
+    @UnitOfWork
+    @ApiOperation(value = "Return all versions in an entry",  hidden = true)
+    @Operation(operationId = "getEntrysVersions", description = "Return all versions in an entry", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    public Set<? extends Version> getEntrysVersions(@Parameter(hidden = true, name = "user")@Auth Optional<User> user,
```

Comment:
I would call it getEntryVersions, here and in line 192. I know you're using it as a possessive, but it reads odd to me. Feel free to ignore this.